homeassistant/components/apple_tv/__init__.py

```diff
@@ -70,204 +83,263 @@ def ensure_list(value: Union[T, Sequence[T]]) -> Sequence[T]:
     extra=vol.ALLOW_EXTRA,
 )
 
-# Currently no attributes but it might change later
-APPLE_TV_SCAN_SCHEMA = vol.Schema({})
-
-APPLE_TV_AUTHENTICATE_SCHEMA = vol.Schema({ATTR_ENTITY_ID: cv.entity_ids})
-
 
-def request_configuration(hass, config, atv, credentials):
-    """Request configuration steps from the user."""
-    configurator = hass.components.configurator
+async def async_setup(hass, config):
+    """Set up the Apple TV integration."""
+    if DOMAIN not in config:
+        return True
+
+    for conf in config[DOMAIN]:
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                DOMAIN, context={"source": config_entries.SOURCE_IMPORT}, data=conf,
+            )
+        )
```

Comment:
Since the configuration is not backwards compatible there is no reason to keep it for imports since yaml configuration for integrations are deprecated.

Reply:
I've been thinking about that as well, but kinda assumed I should continue support it since the integration isn't "new". But it would feel kinda good to remove support for it...

Reply:
Remove it! :)

https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md
"Changes to existing YAML configuration for these same existing integrations, will no longer be accepted."

Reply:
Great, I will kill it with 🔥